packages/remix-dev/package.json

```diff
@@ -17,12 +17,15 @@
   "dependencies": {
     "cacache": "^15.0.5",
     "chokidar": "^3.5.1",
+    "css-tree": "^2.0.4",
```

Comment:
Note: we may not need this afterall, but I'm going to experiment with using the AST to update relative URLs used in CSS. If it's not too much complexity I think we should support that.